src/pages/ConsumerEServiceCatalogPage/components/CatalogCard.tsx

```diff
@@ -31,16 +35,48 @@ export const CatalogCard: React.FC<CatalogCardProps> = ({ eservice }) => {
     )
   }
 
+  const sxCardProps: SxProps =
+    disabled === true
+      ? {
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          minHeight: 410,
+          opacity: 0.5,
+        }
+      : {
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          minHeight: 410,
+        }
```

Comment:
```suggestion
  const sxCardProps: SxProps = {
    height: '100%',
    display: 'flex',
    flexDirection: 'column',
    minHeight: 410,
    opacity: disabled ? 0.5 : 1,
  }
```

Reply:
This could be simplified as such.
Plus, since it is used just once, I would put it "inside" the prop